src/api-service/dev-deploy.ps1

```diff
@@ -0,0 +1,19 @@
+param (
+    [Parameter()]
+    [string] $app_dir="$PSScriptRoot",
+    [Parameter(Mandatory=$true)]
+    [string] $target
+)
+
+Set-Location "$app_dir\..\pytypes"
```

Comment:
Scripts generally shouldn't change the current directory without restoring it, e.g. use Push-Location and in a finally block use Pop-Location.